VENCoreUnitTests/Models/Users/VENUserSpec.m

```diff
@@ -71,21 +71,21 @@
 
 
 describe(@"Initialization", ^{
-
+    
```

Comment:
Ugh. I just `ctrl-I`'ed in Xcode. I guess that's what I get... I'll see if I can clean this up.